src/core/lombok/Version.java

```diff
@@ -0,0 +1,26 @@
+package lombok;
+
+/**
+ * @See VersionableUtils
+ * @author Al3x G0m3z
+ *
+ */
+public enum Version
+{
+    v1,
+    v2,
+    v3,
+    v4,
+    v5,
+    v6,
+    v7,
+    v8,
+    v9,
+    v10,
+    v11,
+    v12,
+    v13,
+    v14,
+    v15,
+    v16
+}

```

Comment:
Thank you for your time guys,
The enum is a bad solution, I take a mistake so the concept of feature is the target, If you are agree with this I will fix them with int for example instead of a String.

Best Regards!
Alex